geopackage_validator/validations/geometry_valid_check.py

```diff
@@ -1,13 +1,16 @@
 from typing import Iterable, Tuple
 
+from geopackage_validator.constants import MAX_VALIDATION_ITERATIONS
 from geopackage_validator.validations import validator
 
+import re
+
 SQL_TEMPLATE = """SELECT 
     ST_IsValidReason("{column_name}") as reason, 
     '{table_name}' as table_name, 
     '{column_name}' as column_name,
     cast(rowid as INTEGER) as row_id
-from "{table_name}" where ST_IsValid("{column_name}") = 0;"""
+from "{table_name}" where ST_IsValid("{column_name}") = 0 LIMIT {limit};"""
```

Comment:
Ook hier zie ik weer dat het `ST_isValid` patroon niet zo goed werkt als in postgis. Ik dacht dat je wel die reason eruit zou kunnen halen, maar dat kan blijkbaar niet. Nu heeft die `LIMIT` het effect dat je de eerste x reasons pakt. en er mogelijk anderssoortige reasons terug kan krijgen, als die later voorkomen. Ook dat is iets om even over na te denken en denk ik ook ff bij Jeroen terug te leggen als we hier geen goede oplosssing voor hebben.

Reply:
Met reason bedoel ik overigens het stuk zonder de detail info, bijv. de POINT waar het optreedt.